src/Windows/Avalonia.Win32/OpenGl/WglDisplay.cs

```diff
@@ -144,8 +142,12 @@ private static void DebugCallback(int source, int type, int id, int severity, in
                             });
                     }
 
-                    using(new WglRestoreContext(dc, context, null))
-                        s_glDebugMessageCallback(Marshal.GetFunctionPointerForDelegate(_debugCallback), IntPtr.Zero);
+                    if (s_glDebugMessageCallback is not null)
+                    {
+                        using (new WglRestoreContext(dc, context, null))
```

Comment:
This `using` is no longer affecting the `return`, is that intentional?

Reply:
It wasn't affecting it before, was it?
I.e. 
```
                    using(new WglRestoreContext(dc, context, null))
                        s_glDebugMessageCallback(Marshal.GetFunctionPointerForDelegate(_debugCallback), IntPtr.Zero);
```
using here was disposed right after s_glDebugMessageCallback exected.
It's still the same, but this context is captured only when s_glDebugMessageCallback is not null

Reply:
Sorry, I read it wrong.